src/D2L.CodeStyle.Analyzers/Language/DefaultValueConsistencyAnalyzer.cs

```diff
@@ -181,7 +181,16 @@ IMethodSymbol implMethod
 				var implDefault = implParameter.ExplicitDefaultValue;
 				var baseDefault = baseParameter.ExplicitDefaultValue;
 
-				if( !implDefault.Equals( baseDefault ) ) {
+				if ( implDefault == null && baseDefault == null ) {

```

Comment:
Opps this was unnecessary stuff beacuse of the `Equals` below, sorry. I first wrote this and realized it should already exist and then found the static method... Will remove